### PR TITLE
[User Config File] English Minor Fixes

### PR DESCRIPTION
**Pull request by EpicSupermind**
Added heading, link and image

### CODE REVIEW

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
+| keyVolumeIncrease  | Key binding to increase universal volume    |
+| keyVolumeDecrease  | Key binding to decrease universal volume    |
+| Skin               | Name of the currently selected skin.        |
+| SkinSamples        | Toggles "Use skin samples" ingame.          |
+| ComboFire          | Toggles Combo Fire on and off. This option cannot be changed ingame.|
+| Precision | Used to toggle "Enable mouse precision" ingame. The option no longer exists, and has been replaced by RawInput.|
+| Width              | Width of your osu! in windowed mode. Change this to set custom window widths.|
+| Height             | Height of your osu! in windowed mode. Change this to set custom window heights. |
+| WidthFullscreen    | Width of your osu! when using fullscreen mode.|
+| HeightFullscreen   | Height of your osu! when using fullscreen mode.|
```
Comment (Shadowtail117) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: Isn't it `osu!client` when referencing the application?

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 2]
```diff
@@ -1,48 +1,52 @@
...
-| Video                        | Toggles "Background Video" ingame.                                                                                                                                                                         |
-| NotifySubmittedThread        | Automatically checks the "Receive email notification on reply" box when submitting your map via BSS.                                                                                                       |
-| AutoChatHide                 | Toggles "Automatically hide chat during gameplay" ingame.                                                                                                                                                  |
+![Combo Fire](CF_DEAD.jpg "Combo Fire")
+
+This a list of options in the osu!< yourpcname >.cfg file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must.
```
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: change `osu!< yourpcname >.cfg` to `` `osu!{yourpcname}.cfg` ``

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 3]
```diff
@@ -1,48 +1,52 @@
...
+
+| Code               | Description                                 |
+|--------------------|---------------------------------------------|
+| TreeSortMode       | Determines the 'group' mode at song select. |
```
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: use code for the code column (do this for this one and the ones below)

**Comment by MegaApplePi** [comment]
line 201: 
```
Hashed value for password. **Do not give this out to others!**
```

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 4]
```diff
@@ -1,230 +1,233 @@
...
-| UsePerBeatmapManiaSpeed      |                                                                                                                                                                                                            |
-| LastVersionPermissionsFailed |                                                                                                                                                                                                            |
-| ManiaSpeedBPMScale           | Forces osu!mania scroll speed to scale with BPM                                                                                                                                                            |
-| MenuTriangles                | Enables triangles                                                                                                                                                                                          |
+This a list of options in the osu!{yourpcname}.cfg file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must.
```
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: put `osu!{yourpcname}.cfg` in code
`` `osu!{yourpcname}.cfg` ``

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 5]
```diff
@@ -1,230 +1,233 @@
...
+| `keyDescreseSpeed`             | Decrease osu!mania scrool speed                                                                                                                                                                            |
+| `Joystick`                     | Enable joystick support                                                                                                                                                                                    |
+| `SaveUsername`                 |                                                                                                                                                                                                            |
+| `SavePassword`                 |                                                                                                                                                                                                            |
+| `KeyOverlay`                   | Display key counter while playing
```
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: remove this

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 6]
```diff
@@ -1,230 +1,232 @@
...
-| UsePerBeatmapManiaSpeed      |                                                                                                                                                                                                            |
-| LastVersionPermissionsFailed |                                                                                                                                                                                                            |
-| ManiaSpeedBPMScale           | Forces osu!mania scroll speed to scale with BPM                                                                                                                                                            |
-| MenuTriangles                | Enables triangles                                                                                                                                                                                          |
-||
+This a list of options in the `osu!{yourpcname}.cfg` file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must.
```
Comment (RaikireHiuduo) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]:
```Markdown
This a list of options in the `osu!{pcName}.cfg` file.
A lot of these options are available through the in-game options and it is recommended to change options there.
Only edit these values manually if requested to.
```

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 7]
```diff
@@ -1,230 +1,232 @@
+# User Configuration File
 
-***[Click to return to Main page](Osu!_Program_Files "wikilink")***
+***[Click to return to Main page](/wiki/osu!_Program_Files)***
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: return to the Main Page?

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 8]
```diff
@@ -1,230 +1,232 @@
...
-| UsePerBeatmapManiaSpeed      |                                                                                                                                                                                                            |
-| LastVersionPermissionsFailed |                                                                                                                                                                                                            |
-| ManiaSpeedBPMScale           | Forces osu!mania scroll speed to scale with BPM                                                                                                                                                            |
-| MenuTriangles                | Enables triangles                                                                                                                                                                                          |
-||
+This a list of options in the `osu!{yourpcname}.cfg` file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must.
+
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: To the table: Add a note that if the Description cell is empty, it uses the description above (e.g. those Editor hotkeys)

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 9]
```diff
@@ -1,230 +1,232 @@
...
-| LastVersionPermissionsFailed |                                                                                                                                                                                                            |
-| ManiaSpeedBPMScale           | Forces osu!mania scroll speed to scale with BPM                                                                                                                                                            |
-| MenuTriangles                | Enables triangles                                                                                                                                                                                          |
-||
+This a list of options in the `osu!{yourpcname}.cfg` file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must. If the `Description` cell is blank, it is the same as the one above it.
```
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: last sentence suggestion:
```
If the `Description` cell is empty, assume it is the same as the one above it.
```

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 10]
```diff
@@ -1,230 +1,232 @@
+# User Configuration File
 
-***[Click to return to Main page](Osu!_Program_Files "wikilink")***
+***[Click to return to Main Page](/wiki/osu!_Program_Files)***
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: Click here to

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 11]
```diff
@@ -1,230 +1,232 @@
...
+| `keyDescreseSpeed`             | Decrease osu!mania scrool speed                                                                                                                                                                            |
+| `Joystick`                     | Enable joystick support                                                                                                                                                                                    |
+| `SaveUsername`                 |                                                                                                                                                                                                            |
+| `SavePassword`                 |                                                                                                                                                                                                            |
+| `KeyOverlay`                   | Display key counter while playing
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: Automatically

Comment (TPGPL) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: capitalize

File: wiki/osu!_Program_Files/User_configuration_file/en.md [hunk 12]
```diff
@@ -1,230 +1,232 @@
...
-| LastVersionPermissionsFailed |                                                                                                                                                                                                            |
-| ManiaSpeedBPMScale           | Forces osu!mania scroll speed to scale with BPM                                                                                                                                                            |
-| MenuTriangles                | Enables triangles                                                                                                                                                                                          |
-||
+This a list of options in the `osu!{yourpcname}.cfg` file. A lot of these options are available through the ingame options and it is recommended that you change options there. Only edit these values manually if you really must. If the `Description` cell is blank, it is the same as the one above it.
+
+| Code                 | Description                                 |
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_configuration_file/en.md]: some Descriptions aren't capitalized, please fix it

File: wiki/osu!_Program_Files/User_Configuration_File/en.md [hunk 13]
```diff
@@ -0,0 +1,232 @@
+# User Configuration File
+
+***[Click here to return to Main Page](/wiki/osu!_Program_Files)***
+
+![Combo Fire](/wiki/shared/combo-fire.jpg "Combo Fire")
```
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_Configuration_File/en.md]: I don't think this image should be here anymore... the feature is long gone and if they want know some trivial things about it, they could just see the glossary

File: wiki/osu!_Program_Files/User_Configuration_File/en.md [hunk 14]
```diff
@@ -0,0 +1,230 @@
+# User Configuration File
+
+***[Click here to return to Main Page](/wiki/osu!_Program_Files)***
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_Configuration_File/en.md]: To the Main Page

File: wiki/osu!_Program_Files/User_Configuration_File/en.md [hunk 15]
```diff
@@ -0,0 +1,230 @@
...
+
+This a list of options in the `osu!{yourpcname}.cfg` file. A lot of these options are available through the in-game options and it is recommended that you change options there. Only edit these values manually if you really must. If the `Description` cell is empty, assume it is the same as the one above it.
```
Comment (TPGPL) [inline wiki/osu!_Program_Files/User_Configuration_File/en.md]: This a list? :thinking:

File: wiki/osu!_Program_Files/User_Configuration_File/en.md (hunk 14 above)
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_Configuration_File/en.md]: use `_Main page: [osu! Program Files](/wiki/osu!_Program_Files)_`

File: wiki/osu!_Program_Files/User_Configuration_File/en.md (hunk 15 above)
Comment (MegaApplePi) [inline wiki/osu!_Program_Files/User_Configuration_File/en.md]: it technically is

Comment (TPGPL) [inline wiki/osu!_Program_Files/User_Configuration_File/en.md]: ?

**Pull request merged**